### DISCUSSION TITLE
weird x: and s: in my code.

### DISCUSSION THREAD

**Discussion by crab-97** [redhat-developer/vscode-java#2757] [Q&A] [answered]
I am getting weird x: in my println line and s: in my print line.
They get automatically written in my code when i run my code.
My code is running fine. And i also am not able to select them or paste anywhere else.
Please help this is my first time learning java.

##### Environment
- Operating System: Windows 10 pro
- JDK version: 19.0.1
- Visual Studio Code version: 1.72.2 (user setup)
                                                Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
                                                Date: 2022-10-12T22:15:18.074Z
                                                Electron: 19.0.17
                                                Chromium: 102.0.5005.167
                                                Node.js: 16.14.2
                                                V8: 10.2.154.15-electron.0
                                                OS: Windows_NT x64 10.0.19045
                                                Sandboxed: No
- Java extension version: v0.25.3

##### Steps To Reproduce
1. Here is my code below
2. 
```java
public class Hello {
   public static void main(String args[]) {
      System.out.println("Hello world");
   }
}
```
3. I just run it and BOOM automatically x: comes in println line.
![code](https://user-images.githubusercontent.com/116727534/198024482-06804e90-aa9a-4525-884a-a12d5b6fe814.png)

**Comment by fbricon** [comment, marked as answer]
These are called inlay hints. Inlay hints add additional inline information to source code to help you understand what the code does.
Parameter name inlay hints show the names of parameters in function calls.

You can toggle inlay hints on/off globally by configuring `editor.inlayHints.enabled`. Other options are:
• `offUnlessPressed` — Hold ctrl+alt / ctrl+option to see inlay hints
• `onUnlessPressed` — Hold ctrl+alt / ctrl+option to hide inlay hints

Specifically for Java, you can configure `java.inlayHints.parameterNames.enabled`: Enable/disable inlay hints for parameter names. Supported values are: 
- `none`(disable parameter name hints), 
- `literals`(Enable parameter name hints only for literal arguments),
- `all`(Enable parameter name hints for literal and non-literal arguments). 

Defaults to `literals`.

**Reply by crab-97** [reply]
Thank you very much.

**Reply by LuisMatos27** [reply]
Thanks  !!!its not about screws its about what particular screw!!!
